### PR TITLE
[WIP] Updating plotly

### PR DESCRIPTION
**Pull request by xmnlab**
this PR is a WIP. This PR tries to check if the new plotly will be better than altair for AlertaDengue.

### CODE REVIEW

**Comment by xmnlab** [comment]
@esloch if you want you can try to move this PR forward :)

**Comment by xmnlab** [comment]
v4.0.0 released! https://github.com/plotly/plotly.py/releases

**Comment by fccoelho** [comment]
@esloch  e @xmnlab, I think the criteria for deciding between the two libraries should be performance and maitainability of the code. Since our site has many plotly visualization already in place, it should be relatively easy to compare both solutions.

**Comment by esloch** [comment]
this new version plotly 4rc is very good! its posible use offline mode only. Altair has its advantages, but plotly is more mature for web applications...

**Comment by esloch** [comment]
we are workin in the upgrade, now using plotly.graph_objs (.go) and graphics plotly.express (.px) some adjustments are still missing, such as assigning color customizations, labels, and other (y) axis in the incident graphs for finish.

**Comment by fccoelho** [comment]
@esloch  we can put this new version in production as soon as this PR is validated. This way we can ask for usability feedback from other users.

File: AlertaDengue/environment.yml [hunk 1]
```diff
@@ -28,9 +29,6 @@ dependencies:
...
 - geopandas
 - sh
-- cufflinks-py
 - ipython
 - sqlparse
-- pip
-- pip:
-    - plotly==2.5.1
\ No newline at end of file
+- plotly=4.0.0rc1
```
Comment (fccoelho) [inline AlertaDengue/environment.yml]: Shouldn't this be `plotly==4.0.0rc1` ?

Comment (xmnlab) [reply AlertaDengue/environment.yml]: IIRC `=` works for conda packages ... and for packages inside pip section it should be `==`

**Comment by xmnlab** [comment]
@esloch I will close this PR .. please create a new one to follow these changes.

**Pull request closed without merging**